api/src/main/scala/org/alephium/api/Endpoints.scala

```diff
@@ -362,6 +362,15 @@ trait Endpoints
       .out(jsonBody[DecodeUnsignedTxResult])
       .summary("Decode an unsigned transaction")
 
+  lazy val getTransaction
+      : BaseEndpoint[(TransactionId, Option[GroupIndex], Option[GroupIndex]), Transaction] =
+    transactionsEndpoint.get
+      .in(path[TransactionId]("txId"))
+      .in(query[Option[GroupIndex]]("fromGroup"))
+      .in(query[Option[GroupIndex]]("toGroup"))
+      .out(jsonBody[Transaction])
+      .summary("Get transaction details")
```

Comment:
did you test the others endpoints still works fine?

Sometime the route mapping can fail as we have other endpoints like:
```
/transactions/status/<tx_id>
/transactions/local-status/<tx_id>
```
calling those endpoint might fail because it will try to decode `status` as a `TransactionId`. 
If it's the case we could go with 
```
/transactions/details/<tx_id>
```

Reply:
Yes, I tested, and other transaction endpoints work fine. But `/transactions/details` is a good idea, will update it, thanks!